content/en/tracing/profiling/_index.md

```diff
@@ -118,12 +118,14 @@ The Datadog Profiler requires Python 2.7+. Memory profiling only works on Python
 | ------------------------------------------------ | ------------- | ------------------------------------------------------------------------------------------------ |
 | `DD_PROFILING_API_KEY`                           | String        | The [Datadog API key][1] to use when uploading profiles.                                         |
 | `DD_PROFILING_TAGS`                              | String        | The tags to apply to an uploaded profile. Must be a list of in the format `<KEY1>:<VALUE1>,<KEY2>:<VALUE2>`.       |
-| `DD_SERVICE_NAME`                                | String        | The Datadog [service][3] name, which can be set here, or in `DD_PROFILING_TAGS`.                    |
-
+| `DD_SERVICE`                                     | String        | The Datadog [service][3] name, which can be set here, or in `DD_PROFILING_TAGS`.                 |
```

Comment:
```suggestion
| `DD_SERVICE`                                     | String        | The Datadog [service][3] name, for example `my-web-app`.                 |
```

Why give them 2 options? How will they know which to choose? Same below.

Reply:
Agreed, and I would put DD_PROFILING_TAGS at the end of the table vs these ones